src/forms/helpers/dropdown-select.js

```diff
@@ -15,7 +15,7 @@ const defaultProps = {
   selectedValues: [],
 }
 
-// Wraps the `DropdownCheckboxGroup` component
+// Wraps the `CheckboxGroup` component: this is only when dropdown is true.
```

Comment:
🍔 I think this would read more naturally as "Wraps the `CheckboxGroup` component when `dropdown` is set to true."